apps/src/lib/node/ledger/shell/process_proposal.rs

```diff
@@ -166,14 +166,23 @@ where
                                     .into(),
                             }
                         } else {
-                            TxResult {
-                                code: ErrorCodes::InvalidVoteExntension.into(),
-                                info: "Process proposal rejected this \
-                                       proposal because the backing stake of \
-                                       the vote extensions published in the \
-                                       proposal was insufficient"
-                                    .into(),
-                            }
+                            // ```ignore
+                            // TxResult {
+                            //     code: ErrorCodes::InvalidVoteExntension. into(),
+                            //     info: " Process proposal rejected this \
+                            //            proposal because the backing stake of \
+                            //            the vote extensions published in the \
+                            //            proposal was insufficient"
+                            //         .into(),
+                            // }
+                            // ```
+                            //
+                            // TODO: is unreachable really fine?
+                            unreachable!(
+                                "We should never get an insufficient backing \
+                                 stake on vote extensions published in \
+                                 proposals."
+                            )
```

Comment:
I'm not sure if this `unreachable!()` condition is fine here. I guess the leader might censor some `VoteExtension` instances here, and if so, I need to get rid of this, and return the `TxResult` value again...